components/eventing-controller/README.md

```diff
@@ -2,40 +2,46 @@
 
 ## Overview
 
-This component contains controllers for various CustomResourceDefinitions related to Eventing in Kyma. The following controller comes with this container:
+The Eventing Controller component contains controllers for various CRDs related to Eventing in Kyma. The following controller comes with this container:
```

Comment:
Or is it

```suggestion
The Eventing component contains controllers for various CRDs related to Eventing in Kyma. The following controller comes with this container:
```

Reply:
We have two components:
the eventing controller, and the event-publisher-proxy

the eventing controller itself contains multiple controllers for various crds...

id say the `The eventing controller components ...` is the way to go